coalib/bears/requirements/GoRequirement.py

```diff
@@ -29,3 +30,23 @@ def __init__(self, package, version="", flag=""):
         """
         PackageRequirement.__init__(self, 'go', package, version)
         self.flag = flag
+
+    def install_command(self):
+        """
+       Creates the installation command for the instance of the class.
+
+       >>> GoRequirement
+       ('github.com/golang/lint/golint', '' , '-u' ).install_command()
```

Comment:
this is not good either. to move stuff on the next line from the same command, you start the command with ``>>>`` in the front, and continue with ``...``, as in:

``>>> a = ( a + 
  ... 5)``